internal/testutil/acc/tpf_converter_test.go

```diff
@@ -67,6 +109,31 @@ func TestConvertAdvancedClusterToTPF(t *testing.T) {
 				name         = "cluster2"
 				cluster_type = "SHARDED"
 
+
+
+
```

Comment:
All these blank lines necessary?

Reply:
I could remove it in the comparison, but I prefer to leave in that way temporarily until i fix the issue, there is this comment: // TODO: RemoveBlock doesn't remove newline just after the block so an extra line is added